include/gsl/gsl_util

```diff
@@ -74,17 +76,26 @@ private:
 };
 
 // finally() - convenience function to generate a final_action
+//#define TRY_WITH_BUG // DEFINE THIS TO SEE THAT TEST CASE "finally_lambda_lvalue" fails to compile
```

Comment:
In my opinion this #if/#else shouldn't be in the final commit, it should be a comment for the commit.

Reply:
yes, sorry, I should have stated here that my #ifdef was only for temporarily demonstrating the issue.
Of course, it should not be part of the final commit.
I can prepare a cleaner commit if you like.